beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -82,8 +80,8 @@ func newBlocksFetcher(ctx context.Context, cfg *blocksFetcherConfig) *blocksFetc
 		headFetcher:    cfg.headFetcher,
 		p2p:            cfg.p2p,
 		rateLimiter:    rateLimiter,
-		fetchRequests:  make(chan *fetchRequestParams, fetchRequestsBuffer),
-		fetchResponses: make(chan *fetchRequestResponse, fetchRequestsBuffer),
+		fetchRequests:  make(chan *fetchRequestParams, queueMaxPendingRequests),
+		fetchResponses: make(chan *fetchRequestResponse, queueMaxPendingRequests),
```

Comment:
queueMaxPendingRequests maybe better naming as this constant is used to fetchResponses too